src/main/java/com/dawsonsystems/session/MongoManager.java

```diff
@@ -200,6 +201,14 @@ public void setSslTrustStorePem(String sslTrustStorePem) {
     this.sslTrustStorePem = sslTrustStorePem;
   }
 
+  public boolean isSslInvalidHostNameAllowed() {
```

Comment:
Are you making this configurable in case we want to submit a PR to the upstream at some point?

Reply:
I made it configurable just in case someone uses our public fork that they do not accidentally use this setting if they do not intend it to.

I think submitting this upstream is unlikely at this point.